doc/nrf/includes/wifi_credentials_shell.txt

```diff
@@ -0,0 +1,20 @@
+Configuring Wi-Fi access point credentials
+==========================================
+
+This sample uses the :ref:`lib_wifi_credentials` library to manage Wi-Fi credentials.
+Before the sample can connect to a Wi-Fi network, you must configure at least one credential set.
+
+Once your device has been flashed with this sample, you can add a credential by connecting to your device's UART interface and then entering the following command:
+
+.. code-block:: console
+
+      wifi_cred add <NetworkSSID> WPA2-PSK <NetworkPassword>
+
+Where ``<NetworkSSID>`` is replaced with the SSID of the Wi-Fi access point you want your device to connect to, and ``<NetworkPassword>`` is its password.
+Then either reboot the device or use the ``wifi_cred auto_connect`` command to manually trigger a connection attempt.
+
+From now on, these credentials will automatically be used when the configured network is reachable.
+
+Enabling :kconfig:option:`CONFIG_WIFI_CREDENTIALS_BACKEND_PSA` instead of using :kconfig:option:`CONFIG_WIFI_CREDENTIALS_BACKEND_SETTINGS` stores Wi-Fi credentials in PSA protected storage, adding more security.
```

Comment:
```suggestion
Enabling the :kconfig:option:`CONFIG_WIFI_CREDENTIALS_BACKEND_PSA` Kconfig option instead of using :kconfig:option:`CONFIG_WIFI_CREDENTIALS_BACKEND_SETTINGS` stores Wi-Fi credentials in a PSA protected storage, adding more security.
```